packages/fxa-auth-server/lib/routes/subscriptions/paypal-notifications.ts

```diff
@@ -31,8 +88,21 @@ export const paypalRoutes = (
     stripeHelper
   );
 
-  // FIXME: All of these need to be wrapped in Stripe error handling
-  // FIXME: Many of these stripe calls need retries with careful thought about
-  //        overall request deadline. Stripe retries must include a idempotency_key.
-  return [];
+  return [
+    {
+      method: 'POST',
+      path: '/oauth/subscriptions/paypal/event',
+      options: {
+        // We'll use the PayPals verification to authenticate.
+        auth: false,
+        // The raw payload is needed for authentication.
+        payload: {
+          output: 'data',
+          parse: false,
+        },
+      },
+      handler: (request: AuthRequest) =>
+        paypalNotificationHandler.handleIpnEvent(request),
```

Comment:
Since `handleIpnEvent` returns a `{}`, should be it returned here as the response content?

Reply:
This will return it, right? An explicit isn't returned isn't needed since its not a block.

Reply:
oh doh, yes, I misread that.  Didn't notice it's a () => expression and not () => block. :-P